Lib/test/test_call.py

```diff
@@ -176,5 +181,102 @@ def test_oldargs1_2_kw(self):
         self.assertRaisesRegex(TypeError, msg, [].count, x=2, y=2)
 
 
+def pyfunc(arg1, arg2):
+    return [arg1, arg2]
+
+
+def pyfunc_noarg():
+    return "noarg"
+
+
+IGNORE_RESULT = object()
+
+
+@cpython_only
+class FastCallTests(unittest.TestCase):
+    # Test calls with positional arguments
+    CALLS_POSARGS = (
+        # (func, args: tuple, result)
+
+        # Python function with 2 arguments
+        (pyfunc, (1, 2), [1, 2]),
+
+        # Python function without argument
+        (pyfunc_noarg, (), "noarg"),
+
+        # C function with aguments
+        (len, ("abc",), 3),
+
+        # C static method
+        ((1).to_bytes, (2, 'little'), b'\x01\x00'),
+
+        # bpo-30524: Test that calling a static method of a type doesn't crash,
+        # ignore the result. The calling convention of datetime.datetime.now()
+        # is METH_FASTCALL | METH_CLASS
+        (datetime.datetime.now, (), IGNORE_RESULT),
+    )
+
+    # Test calls with positional and keyword arguments
+    CALLS_KWARGS = (
+        # (func, args: tuple, kwargs: dict, result)
+
+        # Python function with 2 arguments
+        (pyfunc, (1,), {'arg2': 2}, [1, 2]),
+        (pyfunc, (), {'arg1': 1, 'arg2': 2}, [1, 2]),
+
+        # Python function without argument
+        (pyfunc_noarg, (), {}, "noarg"),
+
+        # C function
+        (max, ([],), {'default': 9}, 9),
+
+        # C static method
+        ((1).to_bytes, (2, 'little'), {}, b'\x01\x00'),
```

Comment:
This case (and the case for pyfunc_noarg) can be removed if add testing `_testcapi.pyobject_fastcalldict(func, args, {})` and `_testcapi.pyobject_fastcallkeywords(func, args, {})` for CALLS_POSARGS.